usmqe/web/tendrl/cluster_work.py

```diff
@@ -0,0 +1,230 @@
+"""
+Description: Simple cluster auxiliary methods and classes
+
+Author: ltrilety
+"""
+
+import pytest
+
+from webstr.selenium.ui.support import WebDriverUtils
+from usmqe.web.tendrl.mainpage.navpage.pages import NavMenuBars
+from usmqe.web.tendrl.mainpage.clusters.cluster_list.pages import\
+    ClustersList, ClustersMenu
+from usmqe.web.tendrl.landing_page.pages import get_landing_page
+from usmqe.web.tendrl.auxiliary.pages import UpperMenu
+from usmqe.web.tendrl.mainpage.clusters.import_cluster_wizard.pages\
+    import ImportCluster
+from usmqe.web.tendrl.mainpage.tasks.pages import TaskDetails
+from usmqe.web.tendrl.mainpage.clusters.cluster.pages import ClusterMenu
+from usmqe.web.tendrl.task_wait import task_wait
+from usmqe.web.tendrl.clusters import check_hosts
+
+
+IMPORT_TIMEOUT = 3600
+
+
+def import_cluster_wait(driver, import_task_details):
+    """
+    wait till the import cluster task is finished
+
+    Parameters:
+        driver: selenium driver
+        import_task_details: webstr page object with import_cluster method
+                       landing_page-HomePage or cluster_list-ClustersMenu
+        ttl (int): how long it waits till the tasks is finished
```

Comment:
Fine, then tell me what the type for selenium driver is? The same for instance of some class? Because that's exactly the type. In other words to put the type here has meaning only if it is basic type.